assets/js/financials.js

```diff
@@ -28,22 +28,15 @@ var munge_data = function(chart_data, chart_name) {
   return data_out;
 };
 
-var raw_data = $('#chart-data').data('chart');
-console.log(raw_data);
-var chart_names = Object.keys(raw_data[0]).filter(function(x) { return x != 'name' });
+var chart_names = ['revenue', 'expenses', 'billable_hours'];
 var charts = [];
 chart_names.map(function(chart_name) {
```

Comment:
Minor, but it's more idiomatic for the function passed to `map` to return values than to mutate some external data structure, like this:
```javascript
var charts = chart_names.map(function(chart_name) {
  var ctx = $('#' + chart_name + '-chart').get(0).getContext('2d');
  var munged_data = munge_data(financial_chart_data, chart_name);
  return Chart(ctx).Bar(munged_data, {
    'scaleShowHorizontalLines': true,
    'scaleShowVerticalLines': true,
    'scaleBeginAtZero': true
  });
});
```